include/deal.II/matrix_free/shape_info.templates.h

```diff
@@ -652,14 +654,22 @@ namespace internal
                                                 scalar_lexicographic,
                                                 lexicographic_numbering);
 
-          subface_interpolation_matrix.resize(fe_1d->n_dofs_per_cell() *
-                                              fe_1d->n_dofs_per_cell());
+          subface_interpolation_matrix_0.resize(fe_1d->n_dofs_per_cell() *
+                                                fe_1d->n_dofs_per_cell());
 
           for (unsigned int i = 0, c = 0; i < fe_1d->n_dofs_per_cell(); ++i)
             for (unsigned int j = 0; j < fe_1d->n_dofs_per_cell(); ++j, ++c)
-              subface_interpolation_matrix[c] =
+              subface_interpolation_matrix_0[c] =
                 interpolation_matrix(scalar_lexicographic[i],
                                      scalar_lexicographic[j]);
+
+          subface_interpolation_matrix_1.resize(
+            subface_interpolation_matrix_0.size());
+
+          for (unsigned int i = 0; i < subface_interpolation_matrix_0.size();
+               ++i)
+            subface_interpolation_matrix_1[i] = subface_interpolation_matrix_0
+              [subface_interpolation_matrix_0.size() - 1 - i];
```

Comment:
I think this would be slightly more general and easier to understand if we requested the other `get_subface_interpolation_matrix` and filled `subface_interpolation_matrix_1` with that matrix. Not that we support anything with non-symmetric shape functions right now, but one never knows where we make extensions in the future.